docs/fastdds/logging/consumer.rst

```diff
@@ -8,20 +8,22 @@ Consumers
 ---------
 
 Consumers are classes that take a |Log::Entry-api| and produce a log output accordingly.
-*eProsima Fast DDS* provides two different log consumers that output log entries to different streams:
+*eProsima Fast DDS* provides three different log consumers that output log entries to different streams:
 
-* :ref:`dds_layer_log_consumer_stdout`: Outputs log entries to STDOUT
-* :ref:`dds_layer_log_consumer_file`: Outputs log entries to a user specified file.
+* :ref:`dds_layer_ostream_consumer_stdout`: Outputs log entries to STDOUT
+* :ref:`dds_layer_ostream_consumer_stdouterr`: Outputs log entries to STDOUT or STDERR depending on the given threshold.
+* :ref:`dds_layer_ostream_consumer_file`: Outputs log entries to a user specified file.
 
 
-.. _dds_layer_log_consumer_stdout:
+.. _dds_layer_ostream_consumer_stdout:
 
 StdoutConsumer
 ^^^^^^^^^^^^^^
 
-|StdoutConsumer-api| is the default log consumer.
-It outputs log entries to STDOUT stream following the convection specified in :ref:`dds_layer_log_logging_spec`.
-By default, the logging module only has the StdoutConsumer.
+|StdoutConsumer-api| outputs log entries to STDOUT stream following the convection specified in
+:ref:`dds_layer_log_logging_spec`.
+It is the default and only log consumer of the logging module if the CMake option |LOG_CONSUMER_DEFAULT| is set to
+``AUTO`` or ``STDOUT``.
```

Comment:
Or not set at all

Reply:
Done in 242f3bf33764025c8523d3cfa206dfa4c22a42f6